app/views/nsm/letters_and_calls/show.html.erb

```diff
@@ -41,16 +41,15 @@
     <% if item.any_adjustments? %>
       <%=
         rows = [
-          [{ text: t('.uplift_allowed'), width: 'govuk-!-width-one-half' }, "#{item.uplift}%"],
           [t('.number_allowed', type: item.type_name), item.count.to_s],
-          [t('.casewoker_allowed'), NumberTo.pounds(item.caseworker_amount)],
+          [{ text: t('.uplift_allowed'), width: 'govuk-!-width-one-half' }, "#{item.uplift}%"],
```

Comment:
Think there's an issue here - when no uplift is applied, we get this: 
<img width="991" alt="image" src="https://github.com/user-attachments/assets/8146a0b5-c785-4a6d-be52-cc4a8b973115" />

This isn't introduced via your code, it was an existing problem - but might be a quick win to fix (default to 0 if item.uplift is nil)